charts/openforms/values.yaml

```diff
@@ -545,6 +545,11 @@ nginx:
     # requests:
     #   cpu: 250m
     #   memory: 256Mi
+  staticFileServing:
+    # -- Enable static file serving directly from Nginx (requires OpenForms >= 3.3.0)
+    enabled: false
+    # -- Path where static files are mounted in the nginx container
+    staticRootVolume: "/srv/static"    
```

Comment:
A minor point, but I wonder if there is a way to clarify that these are settings of our chart and not of the nginx subchart?